www/lobby/id.ts

```diff
@@ -21,7 +44,7 @@ lobby_id_router.get('/:id', async (req: Request, res: Response) => {
   const userId = req.body.id;
```

Comment:
make sure to update this! maybe CTRL-F `req.body.id`

Reply:
got it!